### PR TITLE
Cid 2212

### PR DESCRIPTION
**Pull request by ggallen**

### CODE REVIEW

File: src/main/java/com/redhat/jenkins/plugins/ci/GlobalCIConfiguration.java [hunk 1]
```diff
@@ -104,6 +109,7 @@ protected Object readResolve() {
                     ActiveMqMessagingProvider aconfig = (ActiveMqMessagingProvider) config;
                     if (aconfig.IsMigrationInProgress()) {
                         log.info("Migration in progress for ActiveMqMessagingProvider " + aconfig.getName());
+                        setMigrationInProgress(true);
```
Comment (scoheb) [inline src/main/java/com/redhat/jenkins/plugins/ci/GlobalCIConfiguration.java]: why? it is already true?

Comment (ggallen) [inline src/main/java/com/redhat/jenkins/plugins/ci/GlobalCIConfiguration.java]: It might not be.

Comment (scoheb) [inline src/main/java/com/redhat/jenkins/plugins/ci/GlobalCIConfiguration.java]: but the IF says it is

Comment (ggallen) [inline src/main/java/com/redhat/jenkins/plugins/ci/GlobalCIConfiguration.java]: No, that's a different migrationInProgress variable....

**Pull request merged**